api/services/Flairs.js

```diff
@@ -189,6 +189,26 @@ exports.flairCheck = function (ptrades, svex) {
   }
   var ptradesFlair = "(([0-9]{4}-){2}[0-9]{4})(, (([0-9]{4}-){2}[0-9]{4}))* \\|\\| ([^,|(]*( \\((X|Y|ΩR|αS)(, (X|Y|ΩR|αS))*\\))?)(, ([^,|(]*( \\((X|Y|ΩR|αS)(, (X|Y|ΩR|αS))*\\))?))*";
   var svExFlair = ptradesFlair + " \\|\\| ([0-9]{4}|XXXX)(, (([0-9]{4})|XXXX))*";
+  var tradesParts = ptrades.split("||");
+  var svexParts = svex.split("||");
```

Comment:
Eh, it's probably fine. (I don't think pipes are allowed in in-game IGNs anyway.)

I restarted the travis tests because they errored during npm install, and rerunning the tests has fixed that for me in the past. If it doesn't, let's fix whatever the issue is.